src/pages/RoadSection/RoadSectionPage.test.tsx

```diff
@@ -36,17 +32,14 @@ describe('RoadSectionPage', () => {
     // on render of the page, the error is logged to the console.
     jest.spyOn(console, 'error').mockImplementation(() => {})
 
-    try {
-      // wait until the page is rendered
-      await screen.findByText(/Wegvak niet gevonden/)
-
-      expect(screen.getByText(/Wegvak niet gevonden/)).toBeInTheDocument()
-    } catch (error) {
-      console.error(error)
-    } finally {
-      expect(console.error).toHaveBeenCalled()
-      // @ts-ignore
-      console.error.mockRestore()
-    }
+    // wait until the page is rendered
+    await screen.findByText(/Wegvak niet gevonden/)
+
+    expect(screen.getByText(/Wegvak niet gevonden/)).toBeInTheDocument()
+
+    expect(console.error).toHaveBeenCalled()
```

Comment:
Voelt gek om naar console error te luisteren. Is dat iets dat moet gebeuren? Test de getByText niet hetzelfde mechanisme?

Reply:
Niet mijn code. Geen idee waarom dit zo is bedacht.